src/components/DayPickerRangeController.jsx

```diff
@@ -1005,6 +1006,22 @@ export default class DayPickerRangeController extends React.PureComponent {
     });
   }
 
+  onGetPrevScrollableMonths() {
+    const { numberOfMonths, enableOutsideDays } = this.props;
+    const { currentMonth, visibleDays } = this.state;
+
+    const firstPreviousMonth = currentMonth.clone().subtract(numberOfMonths, 'month');
+    const newVisibleDays = getVisibleDays(firstPreviousMonth, numberOfMonths, enableOutsideDays, true);
+
+    this.setState({
+      currentMonth: firstPreviousMonth.clone(),
```

Comment:
@nkinser Is this why the calendar resets to the farthest date in the past? You're setting the `currentMonth` to the `firstPreviousMonth`

Reply:
@kevinthepan no, `currentMonth` has to be set to `firstPreviousMonth` in order for the previous months to be rendered. I think the reason it shows the earliest month is because when the past months are rendered they push the rest of the months down. I'm not sure how to get around that though